src/txkube/test/test_swagger.py

```diff
@@ -351,15 +353,18 @@ def test_string_date_time(self):
             raises_exception(CheckedValueTypeError),
         )
         now = datetime.utcnow()
+        now_isoformat = now.isoformat()
+        if isinstance(now_isoformat, bytes):
+            now_isoformat = now_isoformat.decode("ascii")
```

Comment:
This came up earlier as well... I guess it's a Python 2/3 difference.  Bytes on Python 2, unicode on Python 3.  Maybe it would be nice to have an isoformat helper that's conditional on Python version and contrives to return the same type on both, relieving all other application code of the burden of this type check.